guidelines/enhancement_template.md

```diff
@@ -88,12 +90,15 @@ implementation details.
 
 Here are some example user stories to show what they might look like:
 
-* As an OpenShift engineer, I want to write an enhancement, so that I can get feedback on my design and 
-build consensus about the approach to take before starting the implementation.
-* As a product manager, I want to review this enhancement proposal, so that I can make sure the customer 
-requirements are met by the design.
-* As an administrator, I want a one-click OpenShift installer, so that I can easily set up a new cluster
-without having to follow a long set of operations.
+* As an OpenShift engineer, I want to write an enhancement, so that I
```

Comment:
Maybe another one: `As an OpenShift engineer, I want to understand the rationale behind a particular feature's design and alternatives considered, so I can work on a new enhancement in that problem space knowing the history of the current design better`

Reply:
Never mind, these were just examples not use cases for the enhancement itself ;)

Reply:
Heh, yeah, but I like the nudge. I'll add it.